### PR TITLE
Birsen

### PR DESCRIPTION
**Pull request by Bir-Eda**
please review my code

### CODE REVIEW

File: src/test/java/features/birsen.feature [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Feature: A description
```
Comment (orhndmrc) [inline src/test/java/features/birsen.feature:1]: Please add javadoc

File: src/test/java/features/birsen.feature [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
```
Comment (orhndmrc) [inline src/test/java/features/birsen.feature]: Please delete empty lines

Comment (Bir-Eda) [reply src/test/java/features/birsen.feature]: I fixed my code please review it again

File: src/test/java/features/birsen.feature (hunk 1 above)
Comment (Bir-Eda) [reply src/test/java/features/birsen.feature:1]: There is no accept in feature class javadoc

**Review by orhndmrc — APPROVED** [review]
They look perfect

**Pull request merged**